itest/rust/src/native_test.rs

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Maybe name the file `native_structures_test`, as "native" has many meanings.

Reply:
:+1: